### ISSUE TITLE
Implementation of initState should start with super.initState()

### ISSUE THREAD

**Issue by LeGoffMael** [dart-lang/lints#78]
**Describe the rule you'd like to see added and to what rule set**
As mentioned in the documentation: https://api.flutter.dev/flutter/widgets/State/initState.html

> Implementations of this method should start with a call to the inherited method, as in super.initState().

So those kind of method should be warned:
```dart
@override
void initState() {
  _controller = AnimationController(vsync: this);
  super.initState();
} 
```
